### PR TITLE
docs(jans-cedarling): add Application Entity back and Update Action Principal and Resources

### LINKED ISSUE #9902: docs(jans-cedarling): fix schema to workload
-------------------

### Description

The Application entity is no longer used, workload is used instead

-------------------

### Prepare

- [x] Read contribution guidelines
- [x] Read license information

------------------

### Identified code changes

N/A

-------------------

### Test cases and code coverage

N/A

-------------------

### Document the changes

N/A

### PR DESCRIPTION
**Pull request by SafinWasi**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  
  
  
  
closes #9902 

#### Implementation Details
  

-------------------
### Test and Document the changes
N/A

Please check the below before submitting your PR. The PR will not be merged if there are no commits that start with `docs:` to indicate documentation changes or if the below checklist is not selected.
- [ ] **I confirm that there is no impact on the docs due to the code changes in this PR.**

### CODE REVIEW

**Review by nynymike — CHANGES_REQUESTED** [review]
Actually... principal could also be Workload.  The resource could be a URL. And perhaps I can get aligned with "Application" as a Type that is specifically used as a resource--not a principal.

File: jans-cedarling/schema/cedarling_core.cedarschema [hunk 1]
```diff
@@ -82,70 +82,74 @@ namespace Jans {
...
+        "name": String,
+        "url": Url,
+    };
 
     // ******  Actions  ******
     action Compare appliesTo {
-        principal: [User, Role],
-        resource: Application,
+        principal: [User, Role, Workload],
+        resource: Workload,
```
Comment (nynymike) [inline jans-cedarling/schema/cedarling_core.cedarschema]: This should be `Application`

File: jans-cedarling/schema/cedarling_core.cedarschema [hunk 2]
```diff
@@ -82,70 +82,74 @@ namespace Jans {
         "header": Set<String>,
         "accept": Set<String>,
     };
+    entity Application = {
+        "name": String,
+        "url": Url,
+    };
```
Comment (nynymike) [inline jans-cedarling/schema/cedarling_core.cedarschema:89]: Let's add `app_id` in case we need a primary key for this entity.

**Review by nynymike — CHANGES_REQUESTED** [review]
resource -> `Application`, and add `app_id` ...

File: jans-cedarling/schema/cedarling_core.cedarschema [hunk 3]
```diff
@@ -82,70 +82,75 @@ namespace Jans {
...
     };
     action Write appliesTo {
-        principal: [User, Role],
+        principal: [User, Role, Workload],
         resource: Application,
         context: Context,
     };
     action GET appliesTo {
-          principal: Workload,
+          principal: [Application, Workload],
```
Comment (nynymike) [inline jans-cedarling/schema/cedarling_core.cedarschema]: The principal cannot be an `Application`

**Review by nynymike — APPROVED** [review]
Great!

**Review by abaghinyan — APPROVED** [review]

**Pull request merged**